initrd/dockerfile.go

```diff
@@ -215,6 +215,43 @@ func (initrd *dockerfile) Name() string {
 	return "Dockerfile"
 }
 
+func startBuildkit(ctx context.Context, buildkitVersion string, port int, printf *testcontainersPrintf) (testcontainers.Container, error) {
+	// Trap any panics that occur when instantiating BuildKit through the
+	// testcontainers library. This is known happen if Docker is not installed.
+	// For more information see:
+	//
+	// https://github.com/unikraft/kraftkit/issues/2001
+	defer func() {
+		if r := recover(); r != nil {
+			log.G(ctx).Warn("Recovered from BuildKit instantiation panic.")
+			log.G(ctx).Warn("This can be caused by Docker missing from the system, or being inaccesible.")
+			log.G(ctx).Warn("")
+			log.G(ctx).Warn("If you think this was caused by something else, please open an issue at:")
+			log.G(ctx).Warn("https://github.com/unikraft/kraftkit/issues")
```

Comment:
```diff
-			log.G(ctx).Warn("Recovered from BuildKit instantiation panic.")
-			log.G(ctx).Warn("This can be caused by Docker missing from the system, or being inaccesible.")
-			log.G(ctx).Warn("")
-			log.G(ctx).Warn("If you think this was caused by something else, please open an issue at:")
-			log.G(ctx).Warn("https://github.com/unikraft/kraftkit/issues")
+			log.G(ctx).Warn("recovered from BuildKit instantiation panic!")
+			log.G(ctx).Warn("this can be caused by Docker missing from the system, or from being inaccesible")
+			log.G(ctx).Warn("")
+			log.G(ctx).Warn("if you think this was caused by something else, please open an issue at:")
+			log.G(ctx).Warn("https://github.com/unikraft/kraftkit/issues")
```